.github/workflows/release-pkg.nu

```diff
@@ -9,19 +9,16 @@
 # Instructions for manually creating an MSI for Winget Releases when they fail
 # Added 2022-11-29 when Windows packaging wouldn't work
 # Updated again on 2023-02-23 because msis are still failing validation
-# Update on 2023-10-18 to use RELEASE_TYPE env var to determine if full or not
 # To run this manual for windows here are the steps I take
 # checkout the release you want to publish
 # 1. git checkout 0.86.0
 # unset CARGO_TARGET_DIR if set (I have to do this in the parent shell to get it to work)
-# 2. $env:CARGO_TARGET_DIR = ""
+# 2. $env.CARGO_TARGET_DIR = ""
```

Comment:
This was actually correct syntax for powershell which is the shell I use as the parent shell mentioned above this line.

Reply:
Oh I see, so that line is in the parent shell, and the rest are in nushell?

Reply:
yup, my comments are pretty terrible here. sry.

Reply:
all good lol